### ISSUE TITLE
Content Engineering Styles Development

### ISSUE THREAD

**Issue by openstaxalina** [openstax/cnx-deploy#1215]
Repos

- [ ]  cnx-recipes v1-41-2
- [ ]  cnx-deploy v3.1.9

**Comment by karenc** [comment]
oer.exports `v0.79.1` was released as part of Products.RhaptosPrint `v0.148` in the last staging release cnx-deploy `v3.1.8`

This dashboard has the versions on different servers that you may find useful: http://qualitas-server.herokuapp.com/dashboards/cnx-repos

cnx-recipes `v1.41.2` is the only change to go out in the next staging release.

**Comment by openstaxalina** [comment]
@karenc thank you!  I was looking in the DevOps channel in Slack for that deploy request for oer.exports v.0.79.1.  I must have missed it.

I've adjusted the card to reflect that only cnx-recipes is getting updated.

Do we need a new cnx-deploy tag (v.3.1.9)?  If so, who creates this?

**Comment by karenc** [comment]
> I was looking in the DevOps channel in Slack for that deploy request/card for oer.exports v.0.79.1. I must have missed it.

@openstaxalina Ah... It didn't go through DevOps because I finally automated it!  So now it just creates a new version of `Products.RhaptosPrint` when `oer.exports` is tagged.  And then I updated the version of `Products.RhaptosPrint` manually and tagged `v3.1.8`.

> Do we need a new cnx-deploy tag (v.3.1.9)? If so, who creates this?

Yes, we need `v3.1.9`.  I've created it! https://github.com/openstax/cnx-deploy/releases/tag/v3.1.9
(The only change is updating `cnx-recipes` to `1.41.2`.)
